servant-client-core/src/Servant/Client/Core/Internal/RunClient.hs

```diff
@@ -19,11 +19,13 @@ import           Servant.API                          (MimeUnrender,
                                                        contentTypes,
                                                        mimeUnrender)
 import           Servant.Client.Core.Internal.Request (Request, Response (..),
+                                                       StreamingResponse (..),
                                                        ServantError (..))
 
 class (Monad m) => RunClient m where
   -- | How to make a request.
   runRequest :: Request -> m Response
+  streamingRequest :: Request -> m StreamingResponse
```

Comment:
The name is a little bit confusing, as it's the response that's streamed to the client, not the request streamed to the server. Don't you think?

Reply:
what name do you suggest?

Reply:
streamedRequest?

Reply:
Hmm. runRequestStreaming, streamingResponse, runStreaming? They all kind of suffer from the "which direction" problem though. So forget what I said. Unless someone comes up with a better suggestion, I'm fine with that name for now =)